packages/gatsby-cli/src/reporter/reporters/index.js

```diff
@@ -0,0 +1,8 @@
+const semver = require(`semver`)
+const { isCI } = require(`ci-info`)
+
+if (semver.satisfies(process.version, `>=8`) && !isCI) {
```

Comment:
ink works on non-interactive consoles, the reason why we check for isCI, is that ink does not pipe to stderr and yurnalist does.

Reply:
Until v3 I guess.

Reply:
I'm not sure why user would want to specifically use `yurnalist` over `ink`? It might make some sense when we start implementing actual CLI changes, but here's mostly 1:1 implementation with only visible changes being different spinner and not mangled output :)